dx_app_code/parliament2/dxapp.json

```diff
@@ -7,7 +7,7 @@
     "WGS"
   ],
   "dxapi": "1.0.0",
-  "version": "0.1.11",
+  "version": "0.1.11a",
```

Comment:
I don't think  0.1.11a follows the [semver spec](https://semver.org/). Is the *a* meant to be a pre-release modifier?